packages/block-library/src/image/index.js

```diff
@@ -96,7 +96,7 @@ const schema = {
 export const settings = {
 	title: __( 'Image' ),
 
-	description: __( 'They’re worth 1,000 words! Insert a single image.' ),
+	description: __( 'Insert an image for visual color.' ),
```

Comment:
Shouldn't this description be a bit more like the Video one? Same goes for the Audio description.